src/pepr/operator/controllers/network/generate.ts

```diff
@@ -23,7 +23,7 @@ export function generate(namespace: string, policy: Allow): kind.NetworkPolicy {
     spec: {
       policyTypes: [policy.direction],
       podSelector: {
-        matchLabels: policy.podLabels,
+        matchLabels: policy.selector,
```

Comment:
Might be missing somewhere else this is happening but do we need/want to still support the deprecated `podLabels` + `remotePodLabels` here? Looks like they were dropped entirely from the logic.

Alternatively I'm okay with dropping them but we'd want to remove from the CRD as well.